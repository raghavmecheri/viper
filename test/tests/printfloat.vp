print(3.0);
print(38264.1);
print(0.0);
print(-99.9);
