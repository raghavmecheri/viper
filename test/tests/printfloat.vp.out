3
38264.1
0
-99.9